packages/experimental/Button/src/ButtonTokens.ts

```diff
@@ -25,6 +28,9 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
       padding: globalTokens.spacing.xs - globalTokens.stroke.width.thin,
       borderWidth: globalTokens.stroke.width.thin,
       iconSize: 16,
+      focused: {
+        borderWidth: 0,
+      },
```

Comment:
Effectively making the border disappear? Would this affect the following calculation?
` paddingHorizontal: globalTokens.spacing.l - globalTokens.stroke.width.thin,`

Reply:
It didn't seem to, but I'll change the padding for explicitness